thunder/executors/fa3ex.py

```diff
@@ -37,6 +37,8 @@ def fa3_fwd_impl(
     if not HAS_FA3:
         raise Exception("fa3 not built, cannot use fa3 executor")  # checker should fail before getting here
 
+    # According to https://github.com/Dao-AILab/flash-attention?tab=readme-ov-file#how-to-use-flashattention, softmax_scale is
```

Comment:
nit-picking but it'd be safer to use a permalink

```suggestion
    # According to https://github.com/Dao-AILab/flash-attention/blob/5018ac6/README.md?plain=1#L112, softmax_scale is
```

Reply:
Fixed, thanks